specificatie/persoon.yaml

```diff
@@ -74,6 +80,8 @@ components:
           $ref: 'overlijden.yaml#/components/schemas/OverlijdenBasis'
         verblijfplaats:
           $ref: 'verblijfplaats.yaml#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        adressering:
+          $ref: 'adressering.yaml#/components/schemas/Adressering'
```

Comment:
volgens mij hadden we afgesproken dat in het zoekresultaat wel de adresregels komen, maar niet de aanschrijfwijze enz. (wel volledigeNaam in Naam). Je hebt nu de volledige adressering inclusief alle samengestelde naamvarianten opgenomen.

Reply:
Ik heb adressering verwijderd. in de veblijfplaatsbeperkt had ik al adresregel1 en adresregel2 opgenomen. Dat heb ik nu gedaan door een adresseringBeperkt toe te voegen en die aan de veblijfplaats voor een binnenlands toe te voegen.

Reply:
Ik het adressering (adresseringbeperkt) nu toegevoegd aan PersoonBeperkt.